twine-upload.sh

```diff
@@ -44,6 +44,10 @@ if [[ ${INPUT_VERBOSE,,} != "false" ]] ; then
     TWINE_EXTRA_ARGS="--verbose $TWINE_EXTRA_ARGS"
 fi
 
+if [[ ${INPUT_PRINT_HASH,,} || ${INPUT_VERBOSE,,} != "false" ]] ; then
+    python /app/print-hash.py
```

Comment:
It's better to pass explicit CLI args:
```suggestion
    python /app/print-hash.py "${INPUT_PACKAGES_DIR%%/}"
```